src/backings/structural/array.ts

```diff
@@ -48,6 +48,7 @@ export class BasicArrayStructuralHandler<T extends ArrayLike<unknown>> extends S
     return newValue;
   }
   fromBytes(data: Uint8Array, start: number, end: number): T {
+    super.validateBytes(data, start, end);
```

Comment:
I think all these instances of `super.validateBytes` should be converted to `this.validateBytes`. We'd likely only need to use `super.validateBytes` if we have override `validateBytes` and need to call the super's method, too.

Reply:
yeah it's done